### PR TITLE
Update CompanyNormalization.md

### PR DESCRIPTION
**Pull request by msundlof**

### CODE REVIEW

File: CompanyNormalization.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ The following parameters may be used in constructing a request to the service:
 -        state (optional) : The state component of the provided company's location.
 -        city (optional) : The city component of the provided company's location.
 -        max_results (optional) : The maximum number of results to be returned, between 1 and 10 (inclusive). Defaults to 3.
+-        use_query_classifier (optional) : Filters out results for bad queries. In the case where a nonsense query is provided to the service, no results will will be returned in the response. This value is defualted to true, in order to turn it off users must provide the param with a value of false.
```
Comment (gliu6) [inline CompanyNormalization.md]: there is a redundant "will" in the second line. And I think you need a comma between "turn it off" and "users...."

File: CompanyNormalization.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ The following parameters may be used in constructing a request to the service:
...
+-        filter_by_country (optional) : With this set to false the country parameter only biases results. Setting this value to true will filter results based on country. This param is defaulted to false.
```
Comment (gliu6) [inline CompanyNormalization.md]: Same thing, I would add a comma after "with this set to false".

File: CompanyNormalization.md [hunk 3]
```diff
@@ -19,6 +19,8 @@ The following parameters may be used in constructing a request to the service:
 -        state (optional) : The state component of the provided company's location.
 -        city (optional) : The city component of the provided company's location.
 -        max_results (optional) : The maximum number of results to be returned, between 1 and 10 (inclusive). Defaults to 3.
+-        use_query_classifier (optional) : Filters out results for bad queries. In the case where a nonsense query is provided to the service, no results will be returned in the response. This value is defualted to true, in order to turn it off users must provide the param with a value of false.
```
Comment (thomasmahoney) [inline CompanyNormalization.md]: defaulted*

And like with the deployment email, let's reuse Qiaoling's wording here, as she already did a good job of summarizing the new features. For the second param, please also specify that the "only biases results" behavior is how the service has historically behaved.

**Review by gliu6 — COMMENTED** [review]
👍

**Comment by thomasmahoney** [comment]
:+1:

**Pull request merged**